x-pack/metricbeat/module/meraki/device_health/_meta/fields.yml

```diff
@@ -0,0 +1,299 @@
+- name: device
```

Comment:
Descriptions required for all the fields

Reply:
that might come in the next release 🙃

Reply:
or in the package